python/lsst/meas/extensions/gaap/_gaussianizePsf.py

```diff
@@ -328,7 +328,7 @@ def _solve(self, kernelCellSet, basisList):
             spatialSolution = spatialkv.getKernelSolution()
         except Exception as e:
             self.log.error("ERROR: Unable to calculate psf matching kernel")
-            log.log("TRACE1." + self.log.getName() + "._solve", log.DEBUG, str(e))
+            log.log("TRACE1." + self.log.name + "._solve", log.DEBUG, str(e))
```

Comment:
Is this way of string concatenation fine?

Reply:
It is what is used in other place. The alternative would be:

```python
log.log(f"TRACE1.{self.log.name}._solve", log.DEBUG, str(e))
```
or an alternative approach that might be more consistent with how other messages are logged:
```python
log.getLogger(f"TRACE1.{self.log.name}._solve").debug("%s", e)
```

Reply:
I incorporated the second approach. The unit tests pass. Can I merge it without running Jenkins again? I'm not even sure if this line ever gets triggered anyway.

Reply:
Should be fine to merge.